modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/panels/EditUserInfoPanel.js

```diff
@@ -0,0 +1,662 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDatePicker from '@clayui/date-picker';
+import ClayForm, {ClayInput, ClaySelectWithOption} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import classnames from 'classnames';
+import {openToast} from 'frontend-js-web';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, {
+	useCallback,
+	useContext,
+	useEffect,
+	useRef,
+	useState,
+} from 'react';
+
+import ChartContext from '../ChartContext';
+import {getUserFullNameDefinition, updateUser} from '../data/users';
+import LogoSelector from '../utils/LogoSelector';
+import {
+	ACTION_KEYS,
+	DEFAULT_IMAGE_PATHS_MAP,
+	DEFAULT_USER_ACCOUNT_FULL_NAME_DEFINITION_FIELDS,
+	INFO_PANEL_MODE_MAP,
+	MODEL_TYPE_MAP,
+	SYMBOLS_MAP,
+} from '../utils/constants';
+import {hasPermission} from '../utils/index';
+
+function EditUserInfoPanel({
+	data,
+	namespace,
+	pathImage,
+	selectLogoURL,
+	spritemap,
+	type,
+	updatePanelViewHandler,
+}) {
+	const [userData, setUserData] = useState({
+		...data,
+		errors: {},
+		isValid: true,
+	});
+	const [isLoading, setIsLoading] = useState(false);
+	const {chartInstanceRef} = useContext(ChartContext);
+	const momentLocaleFormatRef = useRef(
+		moment()
+			.locale(Liferay.ThemeDisplay.getLanguageId())
+			.localeData()
+			.longDateFormat('L')
+	);
+	const [userLanguageId, setUserLanguageId] = useState(data.languageId);
+	const [fullNameDefinition, setFullNameDefinition] = useState([]);
+
+	useEffect(() => {
+		getUserFullNameDefinition(userLanguageId).then((data) => {
+			setFullNameDefinition(
+				data?.userAccountFullNameDefinitionFields ||
```

Comment:
unwrap this array

Reply:
as discussed we leave it as it is because returning directly an array we will have the "page wrapper"